### DISCUSSION TITLE
请问有没有办法把默认浏览器设为chromego所使用的那个chrome？

### DISCUSSION THREAD

**Discussion by DannyJRu** [bannedbook/fanqiang#691] [Q&A] [answered]
我在用chromego的时候，有时候点击一个链接比如腾讯邮件的通知，结果还是用没翻墙的chrome打开的，这样很不方便，我电脑已经把默认浏览器设为了chrome，但是能否设置成点击任何链接都用chromego的chrome打开，而不是没翻墙的chrome打开？

**Comment by bannedbook** [comment, marked as answer]
你不要直接点链接，点右键，复制链接

**Reply by DannyJRu** [reply]
谢谢！
